src/i18n/server/en/meta/enMeta.js

```
@@ -142,6 +142,38 @@
     description:
       'A comfortable and spacious universal bag suitable for both walks and bicycle rides',
   },
+  'chevron-lets-go': {
+    title: 'Chevron "Let\'s Go" – cycling patch with Velcro',
+    description: 'The "Let\'s Go" chevron is a stylish way to personalize your bag, backpack, or gear. Pairs well with the "Strava" and "Coffee" chevrons. Easy Velcro attachment.',
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)